File: 828-chalkboard-xor-game/chalkboard-xor-game.java
class Solution {
    public boolean xorGame(int[] nums) {
        int xor = 0;
        for(int i : nums){
            xor ^= i;
        }
        boolean a = (xor == 0), b = nums.length % 2 == 0;
        return a || b;
    }
}

File: 828-chalkboard-xor-game/chalkboard-xor-game.rs
impl Solution {
    pub fn xor_game(nums: Vec<i32>) -> bool {
        let n = nums.len();
        let mut xor = 0;
        for i in nums{
            xor ^= i;
        }
        let a : bool = (xor == 0);
        let b : bool = (n % 2 == 0);
        a || b
    }
}